samples/boards/nrf52/temp/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(hello_world)
```

Comment:
Project name _temp_nrf52_ or similar would be more appropriate.

Reply:
This sample is specific to nrf5 series SoC. This temperature sensor is inside nrf5 series SoC die, and I cannot find a better place. Maybe under "samples/boards/nrf52/" is good enough for now.
What do you think?

Reply:
I think it will confuse the users if move it to "sample/sensors", cause it not work for other SoC.

Reply:
In this particular comment I was saying the CMake project name in `project(hello_world)` should be something like `project(temp_nrf52)`.

As for the location, what you are demonstrating / testing is the sensor from drivers/sensor/nrf5/ and not the features of a particular board. It is embedded in a family of SoCs, but I don't think people will be confused by this and I would expect to find it in samples/sensor/nrf5/. See what the assigned reviewer has to say before making any changes.

Reply:
The project name should still be changed. I think the current location is fine.

Reply:
@mbolivar thanks. For the project name, what's your suggest name?

Reply:
>  I don't think people will be confused by this and I would expect to find it in samples/sensor/nrf5/

@bwitherspoon If you feel strongly it seems easy enough to move, FWIW.